PS6632257b 
PS6632257b 
WO6632257c 
WI66322582 
WO6632259e 
WO6632274c 
WI66322757 
WO663a1abb 
WI663a1b2a 
OP663a1b8f  --- Start of the program Bug_FinalTestV2.py --- 
OP663a1b90 LOGS: center , pressed
OP663a1b90 LOGS: center , released
OP663a1b91 LOGS: center , pressed
OP663a1b91 LOGS: center , released
OP663a1bb9  --- Start of the program Bug_FinalTestV2.py --- 
OP663a1bbb LOGS: center , pressed
OP663a1bbb LOGS: center , released
OP663a1c8b %cd 'C:\UserData\Ludovic\Serieux\Scolaire\8.EPFL\Master\Semestre 4\Projet de semestre\First step\Stage 2\Debuggin-Tasks\_05_JonasHarbarth\Exercise 5 - The ultimate test'
OP663a1c8b %Run Bug_FinalTestV2.py
OP663a1c8b >>> 
OP663a1c8d Traceback (most recent call last):
OP663a1c8d   
OP663a1c8d File "C:\UserData\Ludovic\Serieux\Scolaire\8.EPFL\Master\Semestre 4\Projet de semestre\First step\Stage 2\Debuggin-Tasks\_05_JonasHarbarth\Exercise 5 - The ultimate test\Bug_FinalTestV2.py", line 9, in <module>
OP663a1c8d     @onevent
OP663a1c8d NameError: name 'onevent' is not defined
OP663a1c8d >>> 
RN663a1c8d 
OP663a1c98  --- Start of the program Bug_FinalTestV2.py --- 
OP663a1c99 LOGS: center , pressed
OP663a1c99 LOGS: center , released
WO663a1cb8 
WI663a1cbc 
CP663a1cbf 
PS663a1cc5 
PS663a1cc5 
OP663a1cce  --- Start of the program Bug_FinalTestV2.py --- 
OP663a1ccf LOGS: center , pressed
OP663a1ccf LOGS: center , released
OP663a1cf0  --- Start of the program Bug_FinalTestV2.py --- 
OP663a1cf1 LOGS: center , pressed
OP663a1cf1 LOGS: center , released
OP663a1cf5 LOGS: center , pressed
OP663a1cf5 LOGS: Acc , -7 , 17 , 1
OP663a1cf5 LOGS: center , released
OP663a1cf5 LOGS: center , pressed
OP663a1cf7 LOGS: Acc , -5 , 19 , 1
OP663a1cf7 LOGS: Acc , 1 , 18 , -8
WO663a1d0f 
WI663a1d12 
UD663a1d97 
UD663a1d97 
UD663a1d98 
UD663a1d98 
OP663a2025 %Run Bug_FinalTestV2.py
OP663a2026 Traceback (most recent call last):
OP663a2026   
OP663a2026 File "C:\UserData\Ludovic\Serieux\Scolaire\8.EPFL\Master\Semestre 4\Projet de semestre\First step\Stage 2\Debuggin-Tasks\_05_JonasHarbarth\Exercise 5 - The ultimate test\Bug_FinalTestV2.py", line 9, in <module>
OP663a2026     @onevent
OP663a2026 NameError: name 'onevent' is not defined
OP663a2026 >>> 
RN663a2026 
WO663a2033 
WI663a2035 
OP663a204c Python 3.7.9 (bundled)
OP663a204c >>> 
OP663a204f  --- Start of the program Bug_FinalTestV2.py --- 
OP663a204f 964
OP663a2075  --- Start of the program Bug_FinalTestV2.py --- 
OP663a2075 959
OP663a2075 975
OP663a2075 995
OP663a2075 980
OP663a2075 982
OP663a2076 988
OP663a2076 983
OP663a2076 989
OP663a2076 983
OP663a2076 983
OP663a2076 979
OP663a2076 981
OP663a2076 982
OP663a2076 328
OP663a2092 %Run Bug_FinalTestV2.py
OP663a2092 Traceback (most recent call last):
OP663a2092   
OP663a2092 File "C:\UserData\Ludovic\Serieux\Scolaire\8.EPFL\Master\Semestre 4\Projet de semestre\First step\Stage 2\Debuggin-Tasks\_05_JonasHarbarth\Exercise 5 - The ultimate test\Bug_FinalTestV2.py", line 9, in <module>
OP663a2092     @onevent
OP663a2092 NameError: name 'onevent' is not defined
OP663a2092 >>> 
RN663a2092 
OP663a2098  --- Start of the program Bug_FinalTestV2.py --- 
OP663a20a8 %Run Bug_FinalTestV2.py
OP663a20a9 Traceback (most recent call last):
OP663a20a9   
OP663a20a9 File "C:\UserData\Ludovic\Serieux\Scolaire\8.EPFL\Master\Semestre 4\Projet de semestre\First step\Stage 2\Debuggin-Tasks\_05_JonasHarbarth\Exercise 5 - The ultimate test\Bug_FinalTestV2.py", line 9, in <module>
OP663a20a9     @onevent
OP663a20a9 NameError: name 'onevent' is not defined
OP663a20a9 >>> 
RN663a20a9 
OP663a20ac  --- Start of the program Bug_FinalTestV2.py --- 
